spec.bs

```diff
@@ -18,3 +18,50 @@ Default Biblio Status: current
 Markup Shorthands: markdown yes
 Assume Explicit For: yes
 </pre>
+
+
+<h2 id="advertisements">Advertisements</h2>
+
+An <dfn export>advertisement</dfn>
+is a [=struct=] with the following [=struct/items=]:
+
+* An <dfn for=advertisement>render URL</dfn>. The render URL must have “https” as the scheme.

```

Comment:
I thought these internal data structures were defined with repeated sentences like this: https://html.spec.whatwg.org/#browsing-context
So this line might look like:
`An [=advertisement=] has a <dfn for=render>render</dfn> [=URL=].`
Should we mention the "https" requirement later, when defining joinAdInterestGroup()?

Reply:
I removed the https text.  We'll need to mention it when describing the algorithm for joinAdInterestGroup.